controls/SV-250316.rb

```diff
@@ -66,7 +66,7 @@ module with the following command:
   tag 'container'
 
   only_if('This check applies to RHEL versions 8.0 and 8.1. If the system is RHEL version 8.2 or newer, this check is Not Applicable.', impact: 0.0) {
-    os.release.to_f < 8.2
+    (os.release.to_f) > 8.1
```

Comment:
We need a way to invoke the `cmp` matcher outside of a test to compare the version strings correctly.